java/opendcs/src/main/java/decodes/tsdb/algo/RdbRating.java

```diff
@@ -81,50 +72,37 @@ public void clearTable()
     }
     public Properties getTableProps() { return tableProps; }
 
-//AW:LOCALVARS_END
-
-//AW:OUTPUTS
+    @Output(type = Double.class)
     NamedVariable dep = new NamedVariable("dep", 0);
-    String _outputNames[] = { "dep" };
-//AW:OUTPUTS_END
 
-//AW:PROPERTIES
-    boolean exceedLowerBound = false;
-    String tableDir = "$DECODES_INSTALL_DIR/rdb";
-    boolean exceedUpperBound = false;
-    boolean applyShifts = false;
-    boolean failIfNoTable = false;
-    String interp = "log"; // possibilities are log and linear
-    String filePrefix = "";
-    String fileSuffix = ".rdb";
-    String _propertyNames[] = { "exceedLowerBound", "tableDir", "exceedUpperBound",
-        "applyShifts", "interp",
-        "filePrefix", "fileSuffix", "failIfNoTable" };
-//AW:PROPERTIES_END
+    @PropertySpec(value = "false")boolean exceedLowerBound = false;
+    @PropertySpec(value = "$DECODES_INSTALL_DIR/rdb")String tableDir = "$DECODES_INSTALL_DIR/rdb";
```

Comment:
The `@PropertySpec` here should be above for readability. Notice that the actual variable declarations are a bit... all over the place.

Reply:
Also when it does make sense to put on same line, make sure there's a space after the ).